sub-packages/bionemo-dnadl/src/bionemo/dnadl/tools/genome_editing.py

```diff
@@ -0,0 +1,51 @@
+# SPDX-FileCopyrightText: Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: LicenseRef-Apache2
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import pandas as pd
+
+from bionemo.dnadl.tools.genome_interval import GenomeInterval
+
+
+def create_personal_sequence(ref_sequence: str, interval: GenomeInterval, variants: pd.DataFrame) -> str:
+    """Create a personal sequence by introducing mutations in the reference sequence."""
+    # TO:DO Implement in a more efficient way as to not bottleneck dataloaders
```

Comment:
or make the model bigger? win win haha.

Reply:
I mean all topline models will take a lot more for the forward than this, but then some of the smaller models actually have on par performance so some may start to use them https://openreview.net/pdf?id=pemixr1Isx . this was also previously discussed at some in the PR process. I can combine _create_personal_sequence and _mutate into 1 and make it faster, but that would be incompatible with added indels. I do have an implementation that just keeps track of reference sequence indeces after insertions/deletions. But sounds like you have something else in mind here https://github.com/NVIDIA/bionemo-framework/pull/209#discussion_r1884639355  that I would like to chat to you about.